src/Processing.js

```diff
@@ -8049,6 +8051,13 @@
           j = i % PConstants.SINCOS_LENGTH;
           p.vertex(centerX + cosLUT[j] * hr, centerY + sinLUT[j] * vr);
         }
+
+        if (mode == PConstants.CHORD)
```

Comment:
in JS, this should be `===` (being strict equality, rather than the coercion that `==` allows for. `0 == ""` is true, for instance, because both can coerce to `false`)